### DISCUSSION TITLE
Convert Text into a Curves

### DISCUSSION THREAD

**Discussion by PeterPanino** [SVG-Edit/svgedit#934] [New feature request]
Please add a command to convert the selected Text object into Curves. That would be very helpful, as it would make the SVG independent from the availability of a specific font.

**Comment by cuixiping** [comment]
It's more easier to convert into a base64 encoded png.
It is technically possible to achieve the function you mentioned, with the latest chrome and local font api.
For older browser, you should load font files from cloud client or user upload.

**Comment by jfhenon** [comment]
No plan for this feature.
